src/server/index.ts

```diff
@@ -0,0 +1,7 @@
+import WebApp from "./webApp";
+
+const web = new WebApp();
+web.run(3000);
+
+const web1 = new WebApp();
+web1.run(8080);
```

Comment:
It's better to remove this line and one above, we just need one instance running.